dotnet-desktop-guide/net/wpf/overview/index.md

```diff
@@ -27,7 +27,7 @@ There are two implementations of WPF:
 
     The .NET Framework implementation of WPF that's supported by Visual Studio 2019 and Visual Studio 2017.
 
-    .NET Framework 4 is a Windows-only version of .NET and is considered a Windows Operating System component. This version of WPF is distributed with .NET Framework. For more information about the .NET Framework version of WPF, see [Introduction to WPF for .NET Framework](../../../framework/wpf/introduction-to-wpf.md?view=netframeworkdesktop-4.8&preserve-view=true).
+    .NET Framework 4 is a Windows-only version of .NET and is considered a Windows Operating System component. This version of WPF is distributed with .NET Framework. For more information about the .NET Framework version of WPF, see [Introduction to WPF for .NET Framework](index.md).
```

Comment:
This link is specifically going to .NET 4, so it should be reverted.
```suggestion
    .NET Framework 4 is a Windows-only version of .NET and is considered a Windows Operating System component. This version of WPF is distributed with .NET Framework. For more information about the .NET Framework version of WPF, see [Introduction to WPF for .NET Framework](../../../framework/wpf/introduction-to-wpf.md?view=netframeworkdesktop-4.8&preserve-view=true).
```

Reply:
@adegeo, that the link goes to .NET 4 isn't a reason to not update when there's a redirect to a .NET 6.0 article:

{
    "Redirect": "TwoWay",
    "SourceUrl": "/dotnet/desktop/wpf/introduction-to-wpf?view=netframeworkdesktop-4.8",
    "TargetUrl": "/dotnet/desktop/wpf/overview/?view=netdesktop-6.0"
},

My recommendation is that we change this link back to `index.md` as generated by the tool.